docs/content/guides/developer/advanced/custom-indexer/prod-setup.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Production Setup
+description: Setup instructions for running the indexer, consistent store, and graphql in production.
+keywords: [ indexer, consistent store, graphql, setup, dex ]
+---
+
+## Indexer setup
+
+The indexer consists of multiple pipelines that each read checkpoint data, transform it, and write it to a table. These
+pipelines are grouped together and configured in TOML files. Multiple instances of the indexer run in parallel, each
+configured by its own TOML file. The grouping is determined by pipeline speed because all pipelines in an instance are
+limited by the slowest pipeline in that instance.
+
+### Run command
+
+An indexer instance is run using this command for each of the configuration files listed below:
+```sh
+sui-indexer-alt indexer \
+    --config CONFIG_TOML
+    --remote-store-url REMOTE_STORE_URL
+    --database-url DATABASE_URL
+```
+
+| Config Param       | Description                                   | Additional info                                                    |
+|--------------------|-----------------------------------------------|--------------------------------------------------------------------|
+| `CONFIG_TOML`      | Path to indexer configuration file            |                                                                    |
+| `REMOTE_STORE_URL` | Location of the checkpoint data to be indexed | ([data sources](/concepts/custom-indexing-framework#data-sources)) |
```

Comment:
nit: maybe just add the data sources link in the "Description" seeing as this is the only row that has additional information.